File: src/main/java/com/lord/small_box/dtos/ExcelItemDto.java
package com.lord.small_box.dtos;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ExcelItemDto {

	private long excelItemId;
	
	private long purchaseOrderId;
	
	private String itemDescription;
	
	private String itemMeasureUnit;
	
	private int quantity;
	
}

File: front/small-box/src/app/models/excelItemDto.ts
export class ExcelItemDto{
    excelItemId!:number;
    purchaseOrderItemId!:number;
   itemDescription!:string;
    itemMeasureUnit!:string;
    itemQuantity!:number;
    itemCode!:string;


    constructor(excelItemId:number,purchaseOrderItemId:number,itemCode:string,itemQuantity:number,itemMeasureUnit:string,itemDescription:string){
       this.excelItemId = excelItemId;
        this.purchaseOrderItemId = purchaseOrderItemId;
        this.itemCode= itemCode;
        this.itemQuantity = itemQuantity;
        this.itemMeasureUnit = itemMeasureUnit;
        this.itemDescription = itemDescription;
      
    }
}